dockerfiles/alpine/is/Dockerfile

```diff
@@ -17,7 +17,7 @@
 # ------------------------------------------------------------------------
 
 # set base Docker image to AdoptOpenJDK Alpine Docker image
-FROM adoptopenjdk/openjdk11:jdk-11.0.8_10-alpine
+FROM adoptopenjdk/openjdk11:jdk-11.0.9_11-alpine
```

Comment:
@RidmiR better if we can check the [architecture of the image](https://github.com/wso2/docker-apim/blob/master/dockerfiles/centos/apim/Dockerfile#L20) as well?

@DilanUA please confirm.

Reply:
since we have not used this practice in other simultaneous releases, let's not consider this for this release.